test/app.test.js

```diff
@@ -29,40 +29,53 @@ describe("app", function() {
                     .check.interaction({
                         state: 'states:start',
                         reply: [
-                            'Hi there! What do you want to do?',
-                            '1. Show this menu again',
-                            '2. Exit'
-                        ].join('\n')
+                            'Welcome to Wazimap! What would you like to do?',
+                            '1. Enter a location to query',
+                            '2. Query a random location',
+                            '3. Exit'
+                        ]
+                        .join('\n')
```

Comment:
We usually put `.join('\n')` on the same line as the closing `]` but only because we think it looks nicer. Feel free to change if you agree or to leave as is if it looks better to you as it is now.